CMakeLists.txt

```diff
@@ -352,7 +352,7 @@ ENDIF()
 SET (Boost_USE_MULTITHREADED ON)
 SET (Boost_FIND_QUIETLY ON)
 
-FIND_PACKAGE (Boost 1.54.0 REQUIRED COMPONENTS date_time regex)
+FIND_PACKAGE (Boost 1.54.0 REQUIRED COMPONENTS date_time regex locale)
```

Comment:
Unicode is still a weakness in the standard library, so using boost+ICU makes perfect sense for that. I think Aaron originally pulled in boost::regex, but I don't remember at this point why he wanted it. Unicode support might well have been the reason.